src/client/transporter.js

```diff
@@ -3,7 +3,12 @@
 const {AuthorizationHandler} = require('./authorization-handler');
 const {CustomEventTarget} = require('./custom-event-target');
 const {EndpointChecker} = require('./endpoint-checker');
-const {assembleUrl} = require('./helpers');
+const {assembleUrl, responseHeaderRetrieve, responseBodyJSON, responseGetStatus} = require('./helpers');
+
+let requestApi = null;
+if(['development', 'test'].includes(process.env.NODE_ENV)) {
```

Comment:
given the amount of duplication, maybe we can extract this in its own method. This could be a const set at startup.

I'm also unsure why we need the `test` env in here if we're forcing everything to be development.

```suggestion
if(MOCKED_API) {
```

Reply:
Running through intellij yielded development as default